spring-cloud-data-admin/src/main/resources/bootstrap.yml

```diff
@@ -1,9 +1,33 @@
+server:
```

Comment:
I don't think `bootstrap.yml` is meant to contain props that are actually applicative properties. Its aim is to _bootstrap_, _i.e._ to tell where very early things, like the config server, are.